hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/StateContext.java

```diff
@@ -411,6 +411,14 @@ public void addContainerActionIfAbsent(ContainerAction containerAction) {
     }
   }
 
+  boolean isSamePipelineAction(PipelineAction action1, PipelineAction action2) {
```

Comment:
Although there is only a "Close" pipeline action now, can we add a comment saying that this only works for close pipeline action? The method name seems generic even though it is not.

Reply:
Yes! Good suggestion